src/helpers/validationSchemas.js

```diff
@@ -86,4 +86,13 @@ export default {
     socialId: Joi.string(),
     provider: Joi.string()
   }),
+  Rating: Joi.object().keys({
```

Comment:
This validation schema will not work.  Just keep this: 
```javascript
rating: Joi.number()
      .integer()
      .positive()
      .min(1)
      .max(5)
      .required()
      .label('Rating must be from 1 to 5 ')
  })
```

No need to wrap it in `Joi.obect().keys({...`